pyqtgraph/graphicsItems/GraphicsObject.py

```diff
@@ -2,8 +2,16 @@
 if QT_LIB.startswith('PyQt'):
     from ..Qt import sip
 from .GraphicsItem import GraphicsItem
+from .. import functions as fn
 
 __all__ = ['GraphicsObject']
+DEBUG = False
+
+try: # prepare common definition for slot decorator across PyQt / Pyside:
+    QT_CORE_SLOT = QtCore.pyqtSlot
+except AttributeError:
+    QT_CORE_SLOT = QtCore.Slot
```

Comment:
This should probably be put in `Qt.py` and handled the same way we handle `QtCore.Signal`

Reply:
👍 done
I didn't see that before.
Now `QtCore.Slot` is a library wide alias for `QtCore.PyQtSlot` when running on PyQt; the same way as `QtCore.Signal` is.